zproject/default_settings.py

```diff
@@ -149,10 +149,8 @@
 THUMBNAIL_IMAGES = False
 SENDFILE_BACKEND: Optional[str] = None
 
-# Base URL of the Tornado server
-# We set it to None when running backend tests or populate_db.
-# We override the port number when running frontend tests.
-TORNADO_SERVER: Optional[str] = 'http://127.0.0.1:9800'
+TORNADO_PORTS: List[int] = []
+USING_TORNADO = True
```

Comment:
I wonder if this could be `USING_TORNADO = len(TORNADO_PORTS) > 0`, with `test_extra_settings.py` just setting `TORNADO_PORTS = []`?  Not sure if that's an improvement.

Reply:
Evaluation order is wrong to do that; `test_extra_settings` is loaded last.